ct_doc/doc/installation.dox

```diff
@@ -202,4 +202,45 @@ Build Flag  | Default value | Description
 -DINTEL_C_COMPILER=\<clang-bin> | "/opt/intel/bin/icpc" | Set Intel C compiler binary
 
 
+\section min_ct_use Minimal use of the installed ct library 
```

Comment:
In fact it is a good idea to provide this section! However, it is not true that the headers and libraries are installed in /usr/* when using the ROS build steps. In that case they remain in the catkin workspace devel/install folders!